core/src/components/range/range.tsx

```diff
@@ -96,6 +96,13 @@ export class Range implements ComponentInterface {
    */
   @Prop() name = this.rangeId;
 
+  /**
+   * The text to display as the control's label. Use this over the `label` slot if
+   * you only need plain text. If both this and the slot are used, this property
```

Comment:
~ Suggestion for tweaking the verbiage, but I am also fine with the changes as-is:

> The plain text to display as the control's label. The `label` property will take priority over the `label` slot if both are used.

Reply:
I kept the "plain text" bit as-is since I like the current more explicit wording, but I like the priority part better in yours. 👍